eslzArm/eslzArm.json

```diff
@@ -1192,6 +1263,161 @@
+                },
+                "parameters": {
+                    "enterpriseScaleCompanyPrefix": {
+                        "value": "[parameters('enterpriseScaleCompanyPrefix')]"
+                    },
+                    "telemetryOptOut": {
+                        "value": "[parameters('telemetryOptOut')]"
+                    },
+                    "platformManagementGroup": {
+                        "value": "[variables('mgmtGroups').platform]"
+                    },
+                    "IdentityManagementGroup": {
+                        "value": "[variables('mgmtGroups').identity]"
+                    },
+                    "managementManagementGroup": {
+                        "value": "[variables('mgmtGroups').management]"
+                    },
+                    "connectivityManagementGroup": {
+                        "value": "[variables('mgmtGroups').connectivity]"
+                    },
+                    "LandingZoneManagementGroup": {
+                        "value": "[variables('mgmtGroups').lzs]"
+                    },
+                    "enableAMBAConnectivity": {
+                        "value": "[parameters('enableMonitorConnectivity')]"
+                    },
+                    "enableAMBAIdentity": {
+                        "value": "[parameters('enableMonitorIdentity')]"
+                    },
+                    "enableAMBALandingZone": {
+                        "value": "[parameters('enableMonitorLandingZones')]"
+                    },
+                    "enableAMBAManagement": {
+                        "value": "[parameters('enableMonitorManagement')]"
+                    },
+                    "enableAMBAServiceHealth": {
+                        "value": "[parameters('enableMonitorBaselines')]"
+                    },
+                    "delayCount": {
+                        "value": "[parameters('delayCount')]"
+                    },
+                    "policyAssignmentParametersCommon": {
+                        "value": {
+                            "alzMonitorResourceGroupName": {
+                                "value": "[parameters('monitorAlertsResourceGroup')]"
+                            },
+                            "alzMonitorResourceGroupLocation": {
+                                "value": "[deployment().location]"
+                            }
+                        }
+                    },
+                    "policyAssignmentParametersServiceHealth": {
+                        "value": {
+                            "alzMonitorActionGroupEmail": {
+                                "value": "[parameters('emailContactActionGroup')]"
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        {
+            // Deploying ESLZ monitor custom policies. Note: all policies should eventually be moved to built-in policies and codebase will be reduced
```

Comment:
is this comment correct? especially the built-in comments?

At a minimum lets call out its the AMBA policies in the comment to make it easier to troubleshoot

Reply:
See above

Reply:
Updated